tests/routes/test_messages.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from src import models  # Import your SQLAlchemy models
+
+
+@pytest.mark.asyncio
+async def test_create_message(client, db_session, test_data):
+    test_app, test_user = test_data
+    # Create a test session
+    test_session = models.Session(
+        user_id=test_user.id, location_id="test_location", metadata={}
+    )
+    db_session.add(test_session)
+    await db_session.commit()
+
+    response = client.post(
+        f"/apps/{test_app.id}/users/{test_user.id}/sessions/{test_session.id}/messages",
+        json={
+            "content": "Test message",
+            "is_user": True,
+            "metadata": {"message_key": "message_value"},
+        },
+    )
+    assert response.status_code == 200
+    data = response.json()
+    assert data["content"] == "Test message"
+    assert data["is_user"] is True
+    assert data["metadata"] == {"message_key": "message_value"}
+    assert "id" in data
+
```

Comment:
testing all the 200s but we should also test the failure modes here, missing data, etc.

Reply:
Agree with this. Just going to get this through the door and get the rest of the edge cases in next week's release